docs/source/changelog.md

```diff
@@ -4,6 +4,31 @@
 
 ## [Unreleased]
 
+## 2.0
+
+### 2.0.0 - 2021-11-22
+
+#### Breaking changes
+
+A breaking change was introduced in [#545](https://github.com/jupyterhub/kubespawner/pull/545), making the default value of `allow_privilege_escalation` be `False`. Practically a consequence of this is that a user can't use `sudo` unless `allow_privilege_escalation` is explicitly set to `True`. Technically it means that the JupyterHub user Pod that KubeSpawner creates will have a container with a `securityContext` that has `allowPrivilegeEscalation` set to `false` by default now.
+
+For reference, the following can be read about `allowPrivilegeEscalation` in [Kubernetes official documentation](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/):
+
+> AllowPrivilegeEscalation: Controls whether a process can gain more privileges than its parent process. This bool directly controls whether the `no_new_privs` flag gets set on the container process. AllowPrivilegeEscalation is true always when the container is: 1) run as `Privileged` OR 2) has `CAP_SYS_ADMIN`.
+
+To revert to the previous behavior, set `allow_privilege_escalation` explicitly to `None`.
```

Comment:
What's the difference between `None` and `False`?

Reply:
Then it is not explicitly set, and that means thre k8s clusters default will be used, which is true by default, but could be false via a podsecuritypolicy